roles/tdarr/tasks/main.yml

```diff
@@ -0,0 +1,59 @@
+#########################################################################
+# Title:            Community: Tdarr                                    #
+# Author(s):        Visorak,Migz                                        #
+# URL:              https://github.com/Cloudbox/Community               #
+# Docker Image(s):  haveagitgat/tdarr_aio                               #
+# --                                                                    #
+#         Part of the Cloudbox project: https://cloudbox.works          #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+- name: "Setting CloudFlare DNS Record"
+  include_role:
+    name: cloudflare-dns
+  vars:
+    record: tdarr
+  when: cloudflare_enabled
+
+- name: Stop and remove any existing container
+  docker_container:
+    name: tdarr
+    state: absent
+
+- name: Create tdarr directories
+  file: "path={{ item }} state=directory mode=0775 owner={{ user.name }} group={{ user.name }}"
+  with_items:
+    - /opt/tdarr
+
+- name: Create and start container
+  docker_container:
+    name: tdarr
+    image: haveagitgat/tdarr_aio
+    pull: yes
+    published_ports:
+      - "127.0.0.1:8265:8265"
+    env:
+      TZ: "{{ tz }}"
+      PUID: "{{ uid }}"
+      PGID: "{{ gid }}"
+      VIRTUAL_HOST: "tdarr.{{ user.domain }}"
+      VIRTUAL_PORT: 8265
+      LETSENCRYPT_HOST: "tdarr.{{ user.domain }}"
+      LETSENCRYPT_EMAIL: "{{ user.email }}"
+    volumes:
+      - "/mnt/unionfs/Media:/home/tdarr/media"
+      - "/opt/tdarr/documents:/home/documents/tdarr"
+      - "/opt/tdarr/db:/var/lib/mongodb"
+      - "/dev/shm:/dev/shm"
+      - "/etc/localtime:/etc/localtime:ro"
```

Comment:
This line needs to be removed. Dont need it now that we/you are using TZ variable.